default/scripting/techs/production/VOID_PREDICTION.focs.txt

```diff
@@ -0,0 +1,35 @@
+Tech
+    name = "PRO_VOID_PREDICTION"
+    description = "PRO_VOID_PREDICTION_DESC"
+    short_description = "IMPERIAL_STOCKPILE_SHORT_DESC"
+    category = "PRODUCTION_CATEGORY"
+    researchcost = 500 * [[TECH_COST_MULTIPLIER]]
+    researchturns = 7
+    tags = [ "PEDIA_PRODUCTION_CATEGORY" ]
+    prerequisites = [
+        "LRN_MIND_VOID"
+        "PRO_PREDICTIVE_STOCKPILING"
+    ]
+    effectsgroups = [
+        EffectsGroup
+            scope = And [
+                ProductionCenter
+                OwnedBy empire = Source.Owner
+                NOT Population high = 0
+                Or [
+                    Focus type = "FOCUS_GROWTH"
+                    Focus type = "FOCUS_LOGISTICS"
+                    Focus type = "FOCUS_STOCKPILE"
+                ]
+            ]
+            effects = [
+                SetMaxStockpile value = Value + Target.Population * [[STOCKPILE_PER_POP]] * ( 1
+                        + 10 * (Statistic If Condition = Focus type = "FOCUS_STOCKPILE")
```

Comment:
why dependent on growth or logistics focus?

Reply:
The idea was to make the techs independent of the focus, just work better if you combine them.
If we dont want such a thing, PRO_GENERIC_SUPPLIES should be a prereq.

Reply:
There is a focus specifically for stockpiling, and the logistics and growth focuses are for other purposes. Doesn't make sense to me to have stockpile related effects depend on them.